### ISSUE TITLE
[TF 1.14 API Docs] tf.train.experimental.enable_mixed_precision_graph_rewrite

### ISSUE THREAD

**Issue by tlkh** [tensorflow/tensorflow#29241]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/train/experimental/enable_mixed_precision_graph_rewrite

### Clear description

* Should better explain the graph rewrite algorithm (black/white/grey listed ops) and where to see the list of ops.
* Overall language can be clearer and more precise.
* Minor issues with the formatting.

### Correct links

Links present are all correct.

### Parameters defined

Briefly explain what the value `"dynamic"` (the default value) for `loss_scale` does, and link to the symbol for that.

### Returns defined

Return values are defined properly.

### Raises listed and defined

Exceptions are not listed nor explained.

### Usage example

No usage example, I can provide a simple code snippet.

Additionally, I want to provide a Colab notebook to demonstrate increase in speed without negative impact on accuracy (on CIFAR10 for example).

### Request visuals, if applicable

* Overview of mixed precision process/flow

### Submit a pull request?

Yes, I have submitted a PR. PR is here #29249

**Comment by ymodak** [comment]
Closing this issue since the associated PR has been merged. Thanks!
